p256/point.ml

```diff
@@ -66,6 +68,17 @@ let pp fmt p =
       Format.fprintf fmt "04%a%a" Cstruct_util.pp_hex_le x
         Cstruct_util.pp_hex_le y
 
+let to_cstruct p =
```

Comment:
Can you update `pp` to use `to_cstruct`?

Reply:
done